src/common/wrap/wrap-raf.js

```diff
@@ -5,25 +5,21 @@
 
 // Request Animation Frame wrapper
 import { ee as baseEE } from '../event-emitter/contextual-ee'
-import { createWrapperWithEmitter as wfn } from './wrap-function'
+import { createWrapperWithEmitter as wfn, unwrapFunction } from './wrap-function'
 import { isBrowserScope } from '../util/global-scope'
 
-const wrapped = {}
+const wrapped = {};
+const RAF_NAME = 'requestAnimationFrame';
 
 export function wrapRaf(sharedEE) {
   const ee = scopedEE(sharedEE)
-  if (wrapped[ee.debugId] || !isBrowserScope) return ee; // animation frames inherently tied to window
-  wrapped[ee.debugId] = true
-  var wrapFn = wfn(ee)
+  if (!isBrowserScope || wrapped[ee.debugId]++) // Notice if our wrapping never ran yet, the falsey NaN will not early return; but if it has,
+    return ee;                                  // then we increment the count to track # of feats using this at runtime. (Check the env before wrapping!)
+  wrapped[ee.debugId] = 1;
 
-  var equestAnimationFrame = 'equestAnimationFrame'
+  var wrapFn = wfn(ee)
 
-  wrapFn.inPlace(window, [
```

Comment:
What is the purpose of not checking for these other properties? Have these all aligned to just use `requestAnimationFrame` across all browsers?

Reply:
You're talking about the 'ms' and 'webkit' prefixes, etc., right? If so, then it's similar to page-visibility.js `___hidden`, for really old versions & backwards compat.